Makefile

```
@@ -268,6 +268,7 @@ bundle: $(KUSTOMIZE) $(OPERATOR_SDK) kustomization
 	(cd config/bundle && $(KUSTOMIZE) edit add resource ../../bundle/manifests/submariner.clusterserviceversion.yaml && cd ../../) && \
 	$(KUSTOMIZE) build config/bundle/ --load_restrictor=LoadRestrictionsNone --output bundle/manifests/submariner.clusterserviceversion.yaml && \
 	sed -i -e 's/$$(SHORT_VERSION)/$(SHORT_VERSION)/g' bundle/manifests/submariner.clusterserviceversion.yaml && \
+	sed -i -e 's/$$(VERSION)/$(VERSION)/g' bundle/manifests/submariner.clusterserviceversion.yaml && \
```

```suggestion
	sed -i -e 's/$${VERSION}/$(VERSION)/g' bundle/manifests/submariner.clusterserviceversion.yaml && \
```

Did you mean this? I suspect it'll fail otherwise

---

`$$(VERSION)` is correct, the goal is to replace the token (which is `$(VERSION)`) in the YAML files, not replace a given value with another value.